### PR TITLE
Fix isDirty

### LINKED ISSUE #67: updateTimestamps in Eloquent/Model fails with latest isDirty
The pull request #66 created an issue.

With the latest PR, `isDirty` will now return true for the tests in `Model->updateTimestamps`. This will prevent `Model` from setting timestamps when creating new models. Not good.

### PR DESCRIPTION
**Pull request by torkiljohnsen**
This is not ideal perhaps, but it fixes #67

### CODE REVIEW

**Comment by vinkla** [comment]
Will this work if you update both the model and the translations model at the same time?

**Comment by torkiljohnsen** [comment]
If you update both, then `isDirty` will return `true` anyways, and `finishSave` will run as normal, so yes.

**Comment by vinkla** [comment]
Okay, good! Could you maybe add a test for this?

**Comment by torkiljohnsen** [comment]
I suppose the overridden `isDirty`, when called with attributes, could also check the translated attributes. That way, `isDirty('title')` for instance, would return true, if `title` was a translated attribute.

I'm at home with a fever, shouldn't be making commits to anything right now. Can you take it from here @vinkla?

File: tests/TranslatorTest.php [hunk 1]
```diff
@@ -187,6 +187,13 @@ public function testUpdateTranslationMakesBaseModelDirty()
         $article->title = 'A new title';
         $this->assertTrue($article->isDirty());
     }
+    
+    public function testIsDirtyCallsBaseModelWhenUsedWithAttributes()
+    {
+        $article = Article::first();
+        $article->title = 'A new title';
+        $this->assertFalse($article->isDirty('foo'));
```
Comment (vinkla) [inline tests/TranslatorTest.php]: Maybe you could also add

```php
$this->assertTrue($article->isDirty('title));
```

Comment (torkiljohnsen) [inline tests/TranslatorTest.php]: That was my thought too, but not now. See latest comment.

**Comment by vinkla** [comment]
Sure. I can take it from here! Thanks for all help.

**Pull request merged**